### PR TITLE
First cut at getting some part of the test suite working for CloudABI

### PR DESCRIPTION
**Pull request by EdSchouten**
I am currently working on creating a Docker container for automated CI for CloudABI. Here are some of the trivial changes that need to land to make tests pass.

### CODE REVIEW

File: src/libtest/lib.rs [hunk 1]
```diff
@@ -977,12 +977,9 @@ fn use_color(opts: &TestOpts) -> bool {
     }
 }
 
-#[cfg(target_os = "redox")]
-fn stdout_isatty() -> bool {
-    // FIXME: Implement isatty on Redox
```
Comment (kennytm) [inline src/libtest/lib.rs]: Please retain this FIXME.

**Review by kennytm — CHANGES_REQUESTED** [review]
r=me after fixing the nit.

File: src/libtest/lib.rs (hunk 1 above)
Comment (EdSchouten) [reply src/libtest/lib.rs]: Done!

**Review by kennytm — APPROVED** [review]

**Pull request merged**